themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -58,27 +58,12 @@ VuFind.register('itemStatuses', function ItemStatuses() {
       el.querySelectorAll('.callnumber,.hideIfDetailed,.location').forEach((e) => e.classList.add('hidden'));
       var locationListHTML = "";
       for (var x = 0; x < result.locationList.length; x++) {
+        var status = result.locationList[x].status;
         locationListHTML += '<div class="groupLocation">';
-        if (result.locationList[x].availability) {
-          locationListHTML += '<span class="text-success">'
-            + VuFind.icon("status-available")
-            + result.locationList[x].location
-            + '</span> ';
-        } else if (typeof(result.locationList[x].status_unknown) !== 'undefined'
-          && result.locationList[x].status_unknown
-        ) {
-          if (result.locationList[x].location) {
-            locationListHTML += '<span class="text-warning">'
-              + VuFind.icon("status-unknown")
-              + result.locationList[x].location
-              + '</span> ';
-          }
-        } else {
-          locationListHTML += '<span class="text-danger">'
-            + VuFind.icon("status-unavailable")
-            + result.locationList[x].location
-            + '</span> ';
-        }
+        locationListHTML += '<span class="' + availabilityClasses[status] + '">'
```

Comment:
You should add availabilityClasses to the global declaration at the top of the file to prevent warnings here.